src/brokers/add-broker/components/AddBrokerForm/YamlEditorView.tsx

```diff
@@ -85,11 +78,14 @@ const YamlEditorView: FC<YamlEditorViewProps> = ({
             <CodeEditor
               value={JSON.stringify(fromState.cr, null, '  ')}
```

Comment:
Could be worth it to transform the `json` to yaml here. That way the user would have a consistent feel with other yaml editors.